docs/hippocampus_model.md

```diff
@@ -0,0 +1,35 @@
+# Trained model for hippocampal segmentation
+## Purpose
```

Comment:
Can we please have blank lines before and after headers? Markdown linters will complain otherwise.